docs/app/pages/components/components-sections/date-time-picker/date-time-picker/date-time-picker.component.html

```diff
@@ -113,6 +113,9 @@
     <tr uxd-api-property name="nowBtnText" type="string">
         Defines the text to be displayed in the button used to set the selected time to the current time.
     </tr>
+    <tr uxd-api-property name="startOfWeek" type="WeekDay" defaultValue="WeekDay.Sunday">
+        Defines the day of the week that should appear in the first column.
```

Comment:
Could you note that the `WeekDay` enumeration comes from @angular/common, since the code example doesn't show it?

Reply:
Thanks! Updated the documentation to mention this.